crypto/src/transaction.rs

```diff
@@ -184,6 +184,12 @@ impl From<Transaction> for transaction::Transaction {
     }
 }
 
+impl From<&Transaction> for transaction::Transaction {
```

Comment:
not really specific to this PR, but this naming is a little confusing, maybe we could do something like `use _ as RawTransaction` or `use _ as ProtoTransaction` or something?

Reply:
good idea, took a pass through this file and replaced the imports (now `ProtoTransactionBody`, `ProtoTransaction`, `ProtoFee`)